src/app/components/PairSelector.tsx

```diff
@@ -8,18 +8,42 @@ export function PairSelector() {
     dispatch(selectPairAddress(pairAddress));
   };
   return (
-    <div className="dropdown" id="pair-selector">
-      <label tabIndex={0} className="btn m-1">
-        {pairSelector.name}
+    <div
+      className="dropdown dropdown-start w-full not-prose"
```

Comment:
it would be super cool to try and not specify `not-prose` - we want to be able to have consistent fonts throughout the website

if prose overrides some styles that you want to apply, you can override them once again with `!`, e.g. `!m-0` instead of `m-0`

Reply:
Nice, I'm new to daisyui and tailwind in particular. Didn't know they had an !m-0. I just saw in the css that non-prose was overwriting when I was using my-0.

Reply:
It looks like the colors are a bit off in the theme. The figma says #191B1D which is base-100 in the theme. But the color for the rest of the background items are #1a1c1e. We might want another issue to line up the colors across the dapp.

![image](https://github.com/DeXter-on-Radix/website/assets/3978285/733b2e47-290b-4c1e-b306-d474b0525428)